src/styles/components/_related-components.scss

```diff
@@ -0,0 +1,53 @@
+$breakpoint-phone: 900px;
+
+.rel-component {
+  background-color: map.get($colors, "accent-teal");
+  padding: 6rem 8rem;
```

Comment:
Saw a small issue in mobile that seems to be related to this padding here:
![Screen Shot 2022-10-26 at 1 44 40 PM](https://user-images.githubusercontent.com/63685366/198099137-393877b6-83a9-4f65-84fd-e4ece3564350.png)

Changed the values in the dev tools to test what would work, `2rem` was kind of just a random choice but it looks like the side padding should be smaller in mobile view:
![Screen Shot 2022-10-26 at 1 45 58 PM](https://user-images.githubusercontent.com/63685366/198099542-8f94006b-fd20-4cc4-bfca-e8d5f34b74c2.png)